analytic_functions.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Sep  2 14:54:32 2020
+
+@author: johannesheyl
+"""
+
+import numpy as np
+
+def eggbox_function(x, y):
+    return np.exp((2+np.cos(x/2)*np.cos(y/2))**5)
+
+def rosenbruck_function(x):
+    N = np.asarray(x).shape[0]
```

Comment:
silly comment but do you mind replacing the `N` with `n`? In that way, all the variable names are not in capital letters.